draft-pauly-add-resolver-discovery.md

```diff
@@ -124,26 +129,65 @@ contain the "dohuri" ({{iana}}). The value stored in the parameter
-   svc.example.net.  7200  IN HTTPSSVC 2 svc1.example.net. (
-									   dohuri=https://doh.example.net/dns-query
-									   odohkey="..." )
+   foo.example.com.  7200  IN HTTPSSVC 1 example.com. (
+                           dohuri=https://doh.example.net/dns-query )
 ~~~
 
-Clients SHOULD NOT accept designations for effective top-level domains (eTLDs), such
-as ".com".
+If this record is DNSSEC-signed, clients can immediately create a mapping that indicates the server as a Designated Resolver for the name in the SVCB record.
+
+If this record is not DNSSEC-signed, clients MUST perform other validation to determine that the zone designation is permitted, as described in {{pvd-mutual}}.
+
+## Additional Designation with PvD JSON {#pvd}
+
+A provisioning domain (PvD) defines a coherent set of information that can be used to access a network and resolve names. {{!I-D.ietf-intarea-provisioning-domains}} defines a JSON dictionary format that can be fetched over HTTPS at the well-known URI "/.well-known/pvd".
+
+Designated Resolvers that are DoH servers SHOULD provide a PvD JSON dictionary available at the well-known PvD URI with the path of the DoH server's URI template appended.
+
+For example, the PvD JSON for the DoH server "https://doh.example.net/dns-query" would be available at "https://doh.example.net/.well-known/pvd/dns-query".
+
+Names that are listed in the "dnsZones" key in the JSON dictionary indicate other names that designate the resolver. For each of those domains, clients SHOULD issue an SVCB query to the DoH resolver. If this record confirms the designation and is DNSSEC-signed, clients can create a mapping to designate the resolver. In order to optimize the validation of these domains, servers MAY use HTTP Server Push to deliver the records prior to the request being made.
+
+The key "dohTemplate" is also defined within the JSON dictionary ({{iana}}) to point back to the DoH URI Template itself. This is used for confirming the DoH server when the PvD is discovered locally or during mutual confirmation ({{pvd-mutual}}).
+
+## Mutual Confirmation with PvD JSON {#pvd-mutual}
+
+Designated DoH Resolvers that provide the PvD JSON described in {{pvd}} can also provide information to allow validation of zone designations without DNSSEC.
+
+The JSON dictionary MAY contains a key "trustedNames" that is an array of strings containing domains that can be used for mutual confirmation of resolver designation.
+
+For example, the JSON dictionary retrieved at "https://doh.example.net/.well-known/pvd/dns-query" can contain the following contents:
+
+~~~
+   {
+     "identifier": "doh.example.net.",
+     "dohTemplate": "https://doh.example.net/dns-query",
+     "dnsZones": ["example.com"],
+     "trustedNames": ["example.com"]
+   }
+~~~
+
+This indicates that "example.com" should be treated as a designated domain, but that it can be validated by checking with the "example.com" server rather than using DNSSEC.
```

Comment:
```suggestion
This indicates that "example.com" should be treated as a designated domain, and that it can be validated by checking with the "example.com" server rather than using DNSSEC.
```